apps/web/src/components/Sidebar/index.tsx

```diff
@@ -9,9 +9,9 @@ export default async function Sidebar({
   commitUuid: string
   projectId: number
 }) {
+  const commit = await findCommitByUuid({ projectId, uuid: commitUuid })
```

Comment:
I feel this `findCommit` it's going to be used on several components on the server at the same time. https://github.com/latitude-dev/latitude-llm/blob/main/apps/web/src/app/(private)/_data-access/index.ts#L11C32-L11C37